ppml/trusted-big-data-ml/scala/docker-occlum/run_spark_on_occlum_glibc.sh

```diff
@@ -333,7 +333,7 @@ run_spark_xgboost() {
                 --executor-memory 9G \
                 --driver-memory 2G \
                 /bin/jars/bigdl-dllib-spark_3.1.2-2.1.0-SNAPSHOT.jar \
-                /host/data /host/data/model 2 100 2
+                -i "/host/data" -s "/host/data/model" -t 2 -r 100 -d 2 -w 1
 }
```

Comment:
https://github.com/intel-analytics/BigDL/blob/main/ppml/trusted-big-data-ml/scala/docker-occlum/kubernetes/run_spark_xgboost.sh This file & related doc also need update